src/sync/index.js

```diff
@@ -1,12 +1,11 @@
-const _ = require('lodash');
+import _ from 'lodash';
+import { Qweb3, Contract } from 'qweb3';
```

Comment:
Yeah this is what I meant in last review, however, the way Qweb3 is referenced is not conventional. 
In Qweb3 src/index.js, Qweb3 as the main module, should be exported using "default export", and Contract, as a submodule, is "export", thus here we referencing them by "import Qweb3, {Contract} from 'qweb3'".

The same use pattern as import React, { Component } from 'react';

Reply:
ah I see what you mean. Qweb3 should be the default export. then Contract, Decoder, etc will be component imports.

Reply:
But the way that qweb3 is setup now, `Qweb3` class is entirely independent from `Contract` class. Shouldn't they all be considered individual components?

Reply:
I think you mean they are unrelated. Yeah, to people who use this library they wouldn't know whether components would relate to each other or not. My point is that libraries usually follow a pattern where there is a default export and multiple module exports.

Reply:
👍 fixed